defaults/main.yml

```diff
@@ -46,6 +46,7 @@ kafka_delete_topic_enable: 'false'
 kafka_auto_create_topics: 'true'
 
 kafka_topics: []
+kafka_not_removable_topics: ['KSTREAM-AGGREGATE-STATE-STORE']  # Contains filter
```

Comment:
Maybe this var should live in `vars/main.yml`

Reply:
Is intention to set it here. The idea is to provide/override a list of your own topics and by default this is what we provide :)